website_sale_product_attribute_filter_visibility/views/product_attribute_views.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="attribute_tree_view" model="ir.ui.view">
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.attribute_tree_view"/>
+        <field name="arch" type="xml">
+            <field name="create_variant" position="after">
+                <field name="website_published" widget="boolean_toggle"/>
+            </field>
+        </field>
+    </record>
+
+    <record id="attribute_view_form" model="ir.ui.view">
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.product_attribute_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//form/sheet/group" position="before">
+                <div class="oe_button_box" name="button_box">
+                    <button name="website_publish_button"
```

Comment:
See now how Odoo adds this smart button here.. 
https://github.com/odoo/odoo/blob/800abfc7baa8461f6b7b5ece95fa7700d3cd872a/addons/website_sale/views/product_views.xml#L91